pkg/cmd/run/root.go

```diff
@@ -126,15 +132,24 @@ var runCmd = &cobra.Command{
 			Api      string `yaml:"api"`
 			Endpoint string `yaml:"endpoint"`
 		}
-		apis := []apiendpoint{}
+		apis := make(map[string]string)
+
+		<-time.NewTimer(time.Second * 2).C
```

Comment:
Wait here is to ensure workers in the pool have had time to register.

We could extend the process pool in the CLI to add event listens and the Add/Remove worker methods to be more reactive about this, but would need a good way to update the output re-actively as well.

Reply:
could you add a comment here, saying this?

Reply:
Fixing this at the moment by adding a reactive worker pool to the CLI

Reply:
Waited has been removed and replaced by a reactive worker pool.